tests/storage/cbt/pull_collect_runner.py

```
@@ -0,0 +1,190 @@
+    download_url = (
+        f"{params['data_endpoint']}?x-kubevirt-export-token={params['export_token']}"
+        f"&offset={chunk_offset}&length={chunk_length}"
+    )
+    bash_command = (
+        "set -o pipefail; "
+        f"curl -s -L --fail --cacert {shlex.quote(PULL_CA_CERT_PATH)} "
+        f"{shlex.quote(download_url)} | "
+        f"dd of={shlex.quote(target_file)} oflag=seek_bytes seek={chunk_offset} "
+        f"conv=notrunc status=none if=/dev/stdin"
+    )
+    subprocess.run(["/bin/bash", "-c", bash_command], check=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)